.github/workflows/build.yml

```diff
@@ -71,9 +71,17 @@ jobs:
         run: rm -rf .eggs && pip install -e .
       - name: Run unittests and generate coverage report
         run: |
+          pip install timm
           coverage run --branch --source mmseg -m pytest tests/
           coverage xml
           coverage report -m
+        if: ${{matrix.torch >= '1.5.0'}}
+      - name: Skip timm unittests and generate coverage report
+        run: |
+          coverage run --branch --source mmseg -m pytest tests/ --ignore tests/test_models/test_backbones/test_timm_backbone.py
```

Comment:
Maybe we could undo the ignore

Reply:
TIMM does not support pt1.3

Reply:
`if: ${{matrix.torch < '1.5.0'}}` will skip the timm unittests.